examples/calculator/calculator.cc

```diff
@@ -7,11 +7,8 @@ using namespace std;
 using namespace pegmatite;
 
 
-/**
- * The AST namespace contains the abstract syntax tree for this grammar.  
- */
-namespace AST
```

Comment:
Please can we keep the AST and Parser namespaces?  They won't generate the most efficient code, but they help document the structure of the code (which, for an example, is probably more important).